index-v4.yaml

```diff
@@ -12,7 +12,7 @@ distributions:
     distribution: [bouncy/distribution.yaml]
     distribution_cache: http://repo.ros2.org/rosdistro_cache/bouncy-cache.yaml.gz
     distribution_status: active
-    distribution_type: ros2
+    distribution_type: end-of-life
```

Comment:
Ardent looks a bit different

```
    distribution_status: end-of-life
    distribution_type: ros2
```

Should `distribution_status` have been changed instead?